README.md

```diff
@@ -15,45 +15,95 @@ Requires Rust 1.51.
 
 Create a new temporary project:
 
-* With no additional dependencies:
-    `$ cargo-temp`
-    
-* With multiple dependencies:
-    `$ cargo-temp rand tokio`
-    
-* When specifying a version:
-    `$ cargo-temp anyhow=1.0`
-    * Using the [cargo's comparison requirements][comparison]:
-        `$ cargo-temp anyhow==1.0.13`
+### With no additional dependencies:
+
+```bash
+cargo-temp
+```
+
+### With multiple dependencies:
+
+```bash
+cargo-temp rand tokio
+```
+
+### When specifying a version:
+
+```bash
+cargo-temp anyhow=1.0`
+```
+
+Using the [cargo's comparison requirements][comparison]:
+
+* Exact version:
+  ```bash
+  cargo-temp anyhow==1.0.13
+  ```
+
+* Maximal version:
+  ```bash
+  cargo-temp anyhow=<1.0.2
+  ```
 
 ### Repositories
 
-You can add repositories to your `Cargo.toml`.
+You can add repositories to your *Cargo.toml*.
```

Comment:
Makes more sense to me to use backticks (code like) instead of bold (emphazis). But you do as you like it's your repo :grin:

Reply:
For me backticks represents code, since *Cargo.toml* is a filename its not really code.
But i'm really not "au jus" about the convention on this.
[cargo-edit](https://github.com/killercup/cargo-edit) use backticks so, let's revert

Reply:
Backticks are also used for everything we don't want any format at all.